haas/model.py

```diff
@@ -227,6 +182,31 @@ def get_console(self):
     def get_console_log_filename(self):
         return '/var/run/haas_console_logs/%s.log' % self.ipmi_host
 
+class Obm(AnonModel):
+    """ Obm superclass supporting various drivers 
+        related to out of band management of servers """
+    type = Column(String, nullable=False)
+
+    __mapper_args__ = {
+            'polymorphic_identity': 'new_obmType',
+            'polymorphic_on': type
+            }
+    @staticmethod
+    def validate(kwargs):
```

Comment:
I do not follow what is the context here ?

--Sahil

On Wed, Aug 19, 2015 at 11:43 PM, Ian Denhardt <notifications@github.com>
wrote:

> In haas/model.py
> <https://github.com/CCI-MOC/haas/pull/484#discussion_r37492654>:
>
> > @@ -227,6 +182,31 @@ def get_console(self):
> >      def get_console_log_filename(self):
> >          return '/var/run/haas_console_logs/%s.log' % self.ipmi_host
> >
> > +class Obm(AnonModel):
> > +    """ Obm superclass supporting various drivers
> > +        related to out of band management of servers """
> > +    type = Column(String, nullable=False)
> > +
> > +    __mapper_args__ = {
> > +            'polymorphic_identity': 'new_obmType',
> > +            'polymorphic_on': type
> > +            }
> > +    @staticmethod
> > +    def validate(kwargs):
>
> This is pretty clearly copypasta from the Switch class. If it contained
> actual functionality I'd suggest a mixin, but since it's just docs It might
> make more sense to just put the docs somewhere else and refer to them in
> both places.
>
> @gsilvis <https://github.com/gsilvis>, @henn <https://github.com/henn>,
> thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CCI-MOC/haas/pull/484/files#r37492654>.
>